src/pokemon_debug.c

```diff
@@ -692,11 +824,18 @@ void CB2_Debug_Pokemon(void)
             FreeAllSpritePalettes();
             gReservedSpritePaletteCount = 8;
             ResetAllPicSprites();
+            //BlendPalettes(PALETTES_ALL, 16, RGB_BLACK);
```

Comment:
Is this line that is commented out needed? :eyes:

Reply:
in this case yes for later